recipes/cvplot/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.43.0"
```

`set_property` requires 1.43